psbt_channel_funder.go

```diff
@@ -92,7 +98,8 @@ func (l *LndPbstChannelFunder) OpenChannel(ctx context.Context,
 	// taproot channel, that uses the PSBT funding flow.
 	taprootCommitType := lnrpc.CommitmentType_SIMPLE_TAPROOT
 	openChanStream, errChan, err := l.lnd.Client.OpenChannelStream(
-		ctx, route.NewVertex(&req.PeerPub), req.ChanAmt, 0, true,
+		ctx, route.NewVertex(&req.PeerPub), req.ChanAmt,
+		CustomChannelRemoteReserve, true,
```

Comment:
The field you're setting here is the push amount. Are you sure we want to donate those 1k sats to the remote party?